changelog/1.19.0.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "1.19.0"
+description: "Released on 05/19/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.19.0.
+
+### Features ✨
+
+- web: Resource quotas have been introduced to Organizations. Resource Quotas define
+  maximum resource utilization applied to each member within the Organization they're
+  applied to. Modify them **Manage > Organizations > Edit Organization**.
+- web: Add site and organization workspace as code templates
```

Comment:
1. Needs documentation

Reply:
Is this referring to the nested templates that are now pushed back, or is this something else @Emyrk?

Reply:
I update this clog periodically during the dev cycle. I haven't made an update since we cut, so this will be removed once I do that.

Reply:
That is